mergify_cli/ci_issues/cli.py

```diff
@@ -0,0 +1,82 @@
+import click
+
+from mergify_cli import utils
+from mergify_cli.ci_issues import junit_upload as junit_upload_mod
+
+
+ci_issues = click.Group(
+    "ci-issues",
+    help="Interact with Mergify's CI Issues",
+)
+
+
+@ci_issues.command(help="Upload JUnit XML reports")
+@click.option(
+    "--api-url",
+    "-u",
+    help="URL of the Mergify API",
+    required=True,
+    envvar="MERGIFY_API_SERVER",
+    default="https://api.mergify.com/v1",
+    show_default=True,
+)
+@click.option(
+    "--token",
+    "-t",
+    help="CI Issues Application Key",
+    required=True,
+    envvar="MERGIFY_CI_ISSUES_TOKEN",
+)
+@click.option(
+    "--repository",
+    "-r",
+    help="Repository full name (owner/repo)",
+    required=True,
+    envvar="REPOSITORY",
+)
+@click.option(
+    "--head-sha",
+    "-s",
+    help="Head SHA of the triggered job",
+    required=True,
+    envvar="HEAD_SHA",
```

Comment:
We should go further and load default depending on where the client is run (Github Action or Circle). Otherwise, it means if we add a new CLI option or we want to update an existing one, we have to update the GitHub actions and the Circle CI orb.